models/models/lstm/model.py

```diff
@@ -98,7 +97,8 @@ def prepare_data(self, midi_file: Path) -> tuple[Any, Any]:
         # data = np.array([x[1] for x in midi_input])
         # return data[:data.shape[0]-1, :], data[1:, :]
 
-    @staticmethod
+        return np.array(notes_sequences), np.array(notes_predictions)
+
```

Comment:
Please bring back static attribute

Reply:
done in 52adc5082de141ef06569d664d3a145761a6b67c, sorry and thanks for being cautious!